name-service/program/src/instruction.rs

```diff
@@ -48,10 +48,12 @@ pub enum NameRegistryInstruction {
     ///   * If account class is `Pubkey::default()`:
     ///   0. `[writeable]` Name record to be updated
     ///   1. `[signer]` Account owner
+    ///   2. `[]` Parent name record (optional)
```

Comment:
I think it'll be clearer to add a third case here rather than adding `2. `[]` Parent name record (optional)` to the two existing cases.

When the parent name record is provided, the signer account must always be the parent owner and not "account owner" or "account class"

Reply:
I agree it's clearer this way